frontend/src/i18n.js

```diff
@@ -0,0 +1,25 @@
+import { createI18n } from "vue-i18n";
+
+const i18n = createI18n({
+  legacy: false,
+  locale: "ENG",
+  globalInjection: true,
+  messages: {
+    ENG: {
+      about: "About",
+      volunteers: "Volunteers",
+      contactus: "Contact Us",
+    },
+    FR: {
+      about: "À propos de nous",
```

Comment:
About = A propos
About us = A propos de nous

Reply:
thank you, corrected